CHANGELOG.md

```diff
@@ -28,7 +28,7 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### Features
 
-- **audit:** add lang query paramenter to help urls ([#1794](https://github.com/dequelabs/axe-core/issues/1794)) ([2d1c8a6](https://github.com/dequelabs/axe-core/commit/2d1c8a6))
+- **audit:** add lang query parameter to help urls ([#1794](https://github.com/dequelabs/axe-core/issues/1794)) ([2d1c8a6](https://github.com/dequelabs/axe-core/commit/2d1c8a6))
```

Comment:
Nice catch!

Unfortunately, this file is generated, so this change will likely be reverted once the next release is made.